pkg/kp/kv.go

```diff
@@ -110,7 +91,7 @@ type consulStore struct {
 	podStore podstore.Store
 }
 
-func NewConsulStore(client consulutil.ConsulClient) Store {
+func NewConsulStore(client consulutil.ConsulClient) *consulStore {
```

Comment:
To wit, golint decrees: `    94  53 warning         exported func NewConsulStore returns unexported type *kp.consulStore, which can be annoying to use (go-golint)
`